impl/src/main/java/com/okta/sdk/impl/ds/ResourceFactoryConfig.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2014 Stormpath, Inc.
+ * Modifications Copyright 2018 Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.sdk.impl.ds;
+
+import java.util.Set;
+
+/**
+ * @since 1.1.0
+ */
+public interface ResourceFactoryConfig {
+
+    Set<String> getSupportedPackages();
+}
```

Comment:
You should add an `.editorconfig` file to detect missing or omit newlines. ;)

Reply:
It has one ;) though maybe it isn't configure right? it has: `insert_final_newline=false`